### ISSUE TITLE
Create the Class GMapCircle

### ISSUE THREAD

**Issue by energywave** [judero01col/GMap.NET#93]
Your control is wonderful and I'm using it with great success. But it's missing a big thing: the native possibility to draw a circle. Actually we can draw a polygon using GMapPolygon but not a circle in the same way (specifying center and radius).
There are some workarounds here and there like drawing a polygon with many many edges so that it seems a circle (that's very inefficient if you have to draw many circles...)
Other way is to draw creating a custom marker.
But there should be a native and easy way to draw a circle, that's a very used feature of markers on map, to display the current position using the accuracy, for example.
Thank you in advance to evaluate the request.

**Comment by jhofran** [comment]
Demo.WindowsForms.CustomMarkers.GMapMarkerCircle ??

**Comment by energywave** [comment]
Yes, I've seen it some time after writing this issue when I've cloned the github repo and used it with success. But when you install the NuGet package, like I did, you don't know that GMapMarkerCircle exists... So it would be nice to move it from the demo to the library, so that it would be a normal object you can use from your code when installing the NuGet package. That's my advice, ad least :)
I've found some bugs when adding/removing markers that I enter a condition where the OnRender go in error with invalid coordinates or something similar. If I'll have some spare time I'll try to make a standalone test to catch the operation that generate that problem and in case I'll open a specific issue or if possible I'll fix the problem.
In the meanwhile thank you for this beautiful control :)
